tests/testthat/line_breaks_and_other/curly-in.R

```diff
@@ -41,3 +41,9 @@ test_that(
     # comment
     expect_equal(1 + 1, 2)
   })
+
+
+while (TRUE) { }
+
+while (TRUE)
+  { }
```

Comment:
We also need to get rid of line-breaks, if present (cf. https://github.com/r-lib/styler/issues/1232):

``` r
"while (TRUE) { 

}" -> code

styler::style_text(code)
#> while (TRUE) {
#> 
#> }
```

<sup>Created on 2024-10-06 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

Reply:
Right…

Reply:
Ok, but that seems to imply that we always remove the line breaks inside an empy `{}`, which _breaks_ a bunch of other test cases, see e1455c66c50bb0e69ddb08e0ae0290c59df3ffd3... Do you think that's inline with the style guide?

Reply:
Yes, I think this is what the style guide dictates, and all the new changes look desirable as per the style guide.